doc/caching.rst

```diff
@@ -15,19 +15,26 @@ or simply undoing some changes within the same workspace.
 Configuration
 =============
 
-For now, Dune cache is an opt-in feature. There are three ways to enable it.
-Choose the one that is more convenient for you:
+.. TODO (ElectreAAS): update version number here when releasing
+Since %version%, Dune cache is now an opt-out feature: it is enabled by default.
+There are three ways to disable it. Choose the one that is more convenient for you:
 
-* Add ``(cache enabled)`` to your Dune configuration file
+* Add ``(cache disabled)`` to your Dune configuration file
   (``~/.config/dune/config`` by default).
-* Set the environment variable ``DUNE_CACHE`` to ``enabled``
-* Run Dune with the ``--cache=enabled`` flag.
+* Set the environment variable ``DUNE_CACHE`` to ``disabled``
+* Run Dune with the ``--cache=disabled`` flag.
 
 By default, Dune stores the cache in your ``XDG_CACHE_HOME`` directory on \*nix
 systems and ``%LOCALAPPDATA%\Microsoft\Windows\Temporary Internet Files\dune`` on Windows.
 You can change the default location by setting the environment variable
 ``DUNE_CACHE_ROOT``.
 
+Caching User Rules
+==================
+
+Not all rules are cached by default: user-created ``(rule _)`` actions are not to prevent mistakes.
+You can enable the caching of these rules by setting the environment variable ``DUNE_CACHE_RULES`` to ``enabled``.
+
```

Comment:
to be clear, this is about caching the output of such rules right? Not the rules themselves

Reply:
I don't really know what caching a rule without caching its result would even mean. So yes I believe this is about caching outputs.